### ISSUE TITLE
Makefile has issues with conda in different enviroments.

### ISSUE THREAD

**Issue by cehbrecht** [bird-house/twitcher#68]
See comments in PR #67.

**Comment by cehbrecht** [comment]
@fmigneault If conda env troubles us in the Makefile we could leave it to the user and start the Makefile when a conda env is created (or install into the system):
```
$ conda env create -f environment
$ conda activate twitcher
$ make install
```

@huard would this be ok? Also for wps ...

In addition we may still have a `bootstrap_conda.sh` script if we like.

**Comment by huard** [comment]
Yes, I always thought the automated creation of an environment was a bit intrusive. 

Other suggestion: rename conda_env to "update" in the Makefile.

**Comment by fmigneault** [comment]
I agree with letting the user decide where the installation should be done.

**Comment by cehbrecht** [comment]
fixed by PR #75.
